plugins/widget/plugin.js

```diff
@@ -2503,11 +2503,10 @@
 			editable.attachListener( evtRoot, 'mousedown', function( evt ) {
 				var target = evt.data.getTarget();
 
-				// http://dev.ckeditor.com/ticket/10887 Clicking scrollbar in IE8 will invoke event with empty target object.
-				if ( !target.type )
-					return false;
+				// (http://dev.ckeditor.com/ticket/10887)
+				// (#663)
+				widget = ( target instanceof CKEDITOR.dom.element ) ? widgetsRepo.getByElement( target ) : null;
```

Comment:
I think we should leave the previous IE comment here and add some Chrome info, e.g.:

```
// Clicking scrollbar in Chrome will invoke event with target object of document type (#663).
// In IE8 the target object will be empty (http://dev.ckeditor.com/ticket/10887).
// We need to check if target is a proper element.
widget = ( target instanceof CKEDITOR.dom.element ) ? widgetsRepo.getByElement( target ) : null;
```